manifests/init.pp

```diff
@@ -425,7 +425,7 @@
     }
   }
 
-  if $ssl_crl_check != "false" {
+  if $ssl_crl_check != 'false' {
```

Comment:
I think these can just be 
```puppet
if $ssl_crl_check {
```

Reply:
I don't think so. The enum will always be true, unless it's undef or `false`. In this case we've not `false` as a boolean but the string `'false'` which evaluates to `true`. tl;dr I think the current implementation is correct

Reply:
Oh right. Forgot that it’s an enum.